e2e_samples/parking_sensors/scripts/deploy_infrastructure.sh

```diff
@@ -56,14 +56,17 @@ az group create --name "$resource_group_name" --location "$AZURE_LOCATION" --tag
 # By default, set all KeyVault permission to deployer
 # Retrieve KeyVault User Id
 kv_owner_object_id=$(az ad signed-in-user show --output json | jq -r '.id')
+kv_owner_name=$(az ad user show --id "$kv_owner_object_id" --output json | jq -r '.userPrincipalName')
+echo "KY Owner NAME:" $kv_owner_name
```

Comment:
Should this be "KV Owner NAME" in the prompt?

Reply:
I needed this value to pass into the bicep module - because of the corrections of the dedicated synapse pool. So it was informative, but I can remove it.